### PR TITLE
zos: use stckf builtin for high-res timer

### PR DESCRIPTION
**Pull request by jBarz**
Instead of gettimeofday which is too heavy for a fast
monotonic clock implementation.

### CODE REVIEW

File: src/unix/os390.c [hunk 1]
```diff
@@ -118,9 +119,10 @@ void uv__platform_loop_delete(uv_loop_t* loop) {
 
 
 uint64_t uv__hrtime(uv_clocktype_t type) {
-  struct timeval time;
-  gettimeofday(&time, NULL);
-  return (uint64_t) time.tv_sec * 1e9 + time.tv_usec * 1e3;
+  uint64_t timestamp;
+  __stckf((unsigned long long*)(&timestamp));
```
Comment (bnoordhuis) [inline src/unix/os390.c]: Is the cast actually necessary?  I mean, isn't `uint64_t` already an `unsigned long long`?  It's either superfluous or dangerous.

Comment (jBarz) [inline src/unix/os390.c]: Not necessary. I have removed it.

**Comment by bnoordhuis** [comment]
Overly cautious CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/310/

**Comment by bnoordhuis** [comment]
Landed in 4776195, thanks.

**Pull request closed without merging**